ladybug_geometry/geometry3d/face.py

```diff
@@ -565,32 +566,41 @@ def is_geometrically_equivalent(self, face, tolerance):
             return False
         return True
 
-    def is_centered_adjacent(self, face, tolerance, angle_tolerance):
+    def is_centered_adjacent(self, face, tolerance):
         """Check whether a given face is centered adjacent with this Face.
 
-        Centered adjacency is definied as being coplananar with this face and sharing
-        the same center point to within the tolerance.
+        Centered adjacency is definied as sharing the same center point as this face,
+        sharing the same area, and being next to one another to within the tolerance.
 
         This is useful for identifying matching surfaces when you want to quickly
         solve for adjacency and you are not concerned about false positives in cases
-        where one face does not perfectly match the other in terms of area or vertices.
+        where one face does not perfectly match the other in terms of vertices.
         This means it is good enough for cases where users know how to set up their
         model correctly.
 
         Args:
             face: Another face for which centered adjacency will be tested.
             tolerance: The minimum difference between the coordinate values of two
                 centers at which they can be considered centered adjacent.
-            angle_tolerance: The max angle in radians that the plane normals can
-                differ from one another in order for them to be considered coplanar.
         Returns:
             True if centered adjacent. False if not centered adjacent.
         """
-        if not self.center.is_equivalent(face.center, tolerance):
+        if not self.center.is_equivalent(face.center, tolerance):  # center check
             return False
-        if not self.plane.is_coplanar_tolerance(face.plane, tolerance, angle_tolerance):
+        if not abs(self.area - face.area) <= tolerance:  # area check
             return False
-        return True
+        # construct a ray using this face's normal and a point on this face
+        v1 = self.boundary[-1] - self.boundary[0]
```

Comment:
How is this check different from checking the center points being equivalent?

Reply:
Here's a visual to explain:
![image](https://user-images.githubusercontent.com/5567574/60697243-6e67a200-9eb7-11e9-99a3-a71aca3e4867.png)

You have a perimeter shape and a core shape.  They both share the same center and (while I know it is not the case in the example) they could have the same area.  However shooting a ray from any point on the blue to will never hit the gray.  It's also important to highlight that the point that it is shot from is not the center but rather a point that I am positive is on the shape (thanks in part to the tolerance input).

There are a few other cases that this catches that would otherwise be missed but I think you can see that there's more that needs to be done than just checking that the centers and areas are equal.